sample-docs/kitchen-sink/blocks.rst

```diff
@@ -238,10 +238,10 @@ With line numbers
             "This is an intentionally very long line because I want to make sure that we are handling scrollable code blocks correctly."
         )
 
-With ``none`` highlighting
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+Without highlighting
+~~~~~~~~~~~~~~~~~~~~
 
-.. code-block:: none
+.. code-block::
```

Comment:
```suggestion
.. code-block:: text
```

Sphinx uses a highlighting lexer by default: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-highlight_language

https://pygments.org/docs/lexers/#pygments.lexers.special.TextLexer seems to be what we want here.